src/utils/components/DiskModal/RegistryDiskModal.tsx

```diff
@@ -51,19 +56,21 @@ const RegistryDiskModal: FC<V1DiskModalProps> = ({
   return (
     <FormProvider {...methods}>
       <TabModal
+        onSubmit={() =>
+          handleSubmit(async (data) => submit(data, vm, editDiskName, onSubmit, pvc))()
```

Comment:
better make submit receive a single obj , and not args, then order isn't important